src/sales-channels/asc/order-settings.md

```diff
@@ -74,27 +74,10 @@ _Order Settings_
 
 Magento orders are created for Amazon orders based on the following status and inventory conditions.
 
-### Order Creation without Multi Source Inventory (MSI)
-
-|Fulfillment Channel|Magento Inventory Status|Amazon Order Status|Create Magento Order Setting|Reserve Inventory = Yes|Reserve Inventory = No|
-|---|---|---|---|---|---|
-|FBA|In-stock<br/>Out-of-stock<br/>Do Not Manage|Pending|No|Inventory not reserved|Inventory not reserved|
-|FBA|In-stock<br/>Out-of-stock<br/>Do Not Manage|PendingAvailability|No|Inventory not reserved|Inventory not reserved|
-|FBA|In-stock<br/>Out-of-stock<br/>Do Not Manage|Canceled|No|Inventory not reserved|Inventory not reserved|
-|FBA|In-stock<br/>Out-of-stock<br/>Do Not Manage|Error|No|Inventory not reserved|Inventory not reserved|
-|FBA|In-stock<br/>Out-of-stock<br/>Do Not Manage|Unshipped|No|Inventory not reserved|Inventory not reserved|
-|FBA|In-stock<br/>Out-of-stock<br/>Do Not Manage|PartiallyShipped|No|Inventory not reserved|Inventory not reserved|
-|FBA|In-stock<br/>Out-of-stock<br/>Do Not Manage|Shipped|Yes|Inventory not reserved|Inventory not reserved|
-|FBM|In-stock<br/>Out-of-stock<br/>Do Not Manage|Pending|No|Inventory reserved|Inventory not reserved|
-|FBM|In-stock<br/>Out-of-stock<br/>Do Not Manage|PendingAvailability|No|Inventory not reserved |Inventory not reserved|
-|FBM|In-stock<br/>Out-of-stock<br/>Do Not Manage|Canceled|No|Inventory not reserved|Inventory not reserved|
-|FBM|In-stock<br/>Out-of-stock<br/>Do Not Manage|Error|No|Inventory not reserved|Inventory not reserved|
-|FBM|In-stock<br/>Out-of-stock<br/>Do Not Manage|Unshipped|Yes|Inventory reserved|Inventory not reserved|
-|FBM|In-stock<br/>Out-of-stock<br/>Do Not Manage|PartiallyShipped|Yes|Inventory reserved|Inventory not reserved|
-|FBM|In-stock<br/>Out-of-stock<br/>Do Not Manage|Shipped|Yes|Inventory reserved|Inventory not reserved|
-
 ### Order Creation with Multi Source Inventory (MSI)
 
+*Supported in Magento 2.3.x integrations only.
```

Comment:
The `*` with no space? Not sure what was intended here.

Reply:
it was an attempt at a note, on the fly, doing too many things at once.   changing.